libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/AudioCapabilities.java

```diff
@@ -210,7 +211,8 @@ public Pair<Integer, Integer> getEncodingAndChannelConfigForPassthrough(Format f
       return null;
     }
     int channelCount;
-    if (format.channelCount == Format.NO_VALUE || encoding == C.ENCODING_E_AC3_JOC) {
+    if (format.channelCount == Format.NO_VALUE || encoding == C.ENCODING_E_AC3_JOC
+        || encoding == C.ENCODING_DTS) {
       // In HLS chunkless preparation, the format channel count and sample rate may be unset. See
       // https://github.com/google/ExoPlayer/issues/10204 and b/222127949 for more details.
       // For E-AC3 JOC, the format is object based so the format channel count is arbitrary.
```

Comment:
We need to extend the comment here and explain why we don't care about the container's (compressed bitsteam) channel count for ENCODING_DTS too. 

Also, this is probably *not* true for ENCODING_DTS but one of AUDIO_DTS_EXPRESS or AUDIO_DTS_X, right? This goes back to the other comment that we need to change the logic here to handle AUDIO_DTS_EXPRESS and AUDIO_DTS_X instead of MimeTypes

Reply:
Agree. Gathering the idea from our comments so far, I think this should be something like `format.channelCount == Format.NO_VALUE || encoding == C.ENCODING_E_AC3_JOC || encoding == C.ENCODING_DTS_HD || encoding == C.ENCODING_DTS_UHD_P2` later.

And the comment explaining the new cases is definitely needed here :)

Reply:
@tianyif This change has been reverted. DTS Encoding's format.channelcount should not be skipped. Please see dc85ffb17456fe7cfa6eb135025665800948de1a  Thanks.